doc/nvcounter_syscalls.md

```diff
@@ -0,0 +1,56 @@
+---
+driver number: 0x80040000
+---
+
+NvCounter System Calls
+======================
+
+## Overview
+
+The NvCounter driver provides a global, non-volatile, atomically-incremented,
+anti-rollback counter. The counter must be initialized (to a value of 0) by
+kernel code, but can then be incremented by userspace.
+
+## Command
+
+1.  Read and increment. Callback ID 0 will be called when the increment is
```

Comment:
Should this be down under Command number: 1?

Reply:
Oops, forgot to remove revision 1 after I rewrote the description. Removed.